README.md

```diff
@@ -32,6 +32,7 @@ The following MCP tools are currently implemented:
 | list-pulse-metrics-from-metric-definition-id      | List Pulse Metrics from Metric Definition ID ([Pulse API][pulse])                              |
 | list-pulse-metrics-from-metric-ids                | List Pulse Metrics from Metric IDs ([Pulse API][pulse])                                        |
 | list-pulse-metric-subscriptions                   | List Pulse Metric Subscriptions for Current User ([Pulse API][pulse])                          |
+| generate-pulse-metric-value-insight-bundle        | Generate Pulse Metric Value Insight Bundle ([Pulse API][pulse])                                |
```

Comment:
Hey Brian, thanks for these changes. Could you please share more about the motivation for adding this tool? Where would it fit into a customer's agentic workflow?

Reply:
Oh, I see, thanks. This is just the `retrievePulseInsight` tool scheduled for 260 then. Do you think there is any value in adding a parameter to the tool that would cause it to intelligently choose between the ban, basic, detail, or springboard endpoints instead of just ban?

Reply:
I was starting with the common one and looking to get feedback.  It's not too much to add the rest though, so I've made the changes.  It'll default to 'ban' if they don't specify, and did some additional tests with the changes.

Reply:
Nice, thanks!